### PR TITLE
feat(card): update card info and description in following feed

### PR DESCRIPTION
**Pull request by devformatters**

### CODE REVIEW

File: src/views/Follow/Feed/UserAddArticleTagActivity/index.tsx [hunk 1]
```diff
@@ -16,8 +16,8 @@ const UserAddArticleTagActivity = ({
       <FeedHead>
         <span>
           <Translate
-            zh_hant="添加精選於"
-            zh_hans="添加精选于"
+            zh_hant="追蹤標籤有新作品"
```
Comment (robertu7) [inline src/views/Follow/Feed/UserAddArticleTagActivity/index.tsx:19]: 追蹤標籤有新精選作品？

File: src/components/ArticleDigest/Feed/FooterActions/ReadTime/index.tsx [hunk 2]
```diff
@@ -25,16 +30,30 @@ const ResponseCount = ({ article, hasDate }: ResponseCountProps) => {
 
   return (
     <>
-      <TextIcon icon={<IconReadTime12 size="xs" />} size="xs" color="grey-dark">
-        {numAbbr(readHour, 1)}{' '}
-        <Translate zh_hant="小時" zh_hans="小时" en="hours" />
-      </TextIcon>
-
-      {hasDate && <IconDotDivider />}
+      <Tooltip
```
Comment (robertu7) [inline src/components/ArticleDigest/Feed/FooterActions/ReadTime/index.tsx:33]: may need to measure the impact of performance

**Review by robertu7 — APPROVED** [review]

File: src/views/Follow/Feed/UserAddArticleTagActivity/index.tsx (hunk 1 above)
Comment (devformatters) [reply src/views/Follow/Feed/UserAddArticleTagActivity/index.tsx:19]: Will correct in the following PR.

File: src/components/ArticleDigest/Feed/FooterActions/ReadTime/index.tsx (hunk 2 above)
Comment (devformatters) [reply src/components/ArticleDigest/Feed/FooterActions/ReadTime/index.tsx:33]: Will correct in the following PR.

**Pull request merged**